modules/virt-adding-virtio-drivers-vm-yaml.adoc

```diff
@@ -8,28 +8,24 @@
 [id="virt-adding-virtio-drivers-vm-yaml_{context}"]
 = Adding VirtIO drivers container disk to a virtual machine
 
-{VirtProductName} distributes VirtIO drivers for Microsoft Windows as a
-container disk, which is available from the
-link:https://access.redhat.com/containers/#/registry.access.redhat.com/container-native-virtualization/virtio-win[Red Hat Ecosystem Catalog].
-To install these drivers to a Windows virtual machine, attach the
-`container-native-virtualization/virtio-win` container disk to the virtual machine as a SATA CD drive
-in the virtual machine configuration file.
+Add a VirtIO drivers container disk to a Windows virtual machine (VM) as a SATA CD drive.
 
 .Prerequisites
 
 * Download the `container-native-virtualization/virtio-win` container disk from the
-link:https://access.redhat.com/containers/#/registry.access.redhat.com/container-native-virtualization/virtio-win[Red Hat Ecosystem Catalog].
-This is not mandatory, because the container disk will be downloaded from the Red Hat registry
-if it not already present in the cluster, but it can reduce installation time.
+link:https://catalog.redhat.com/software/containers/search?q=virtio-win&p=1[Red Hat Ecosystem Catalog].
++
+This is not mandatory, because the container disk is downloaded from the Red Hat registry
+if it not already present in the cluster. However, downloading reduces the installation time.
```

Comment:
Could this be a tip instead? It's not exactly a prereq if the procedure will work fine without it.